rusk-wallet/config.toml

```diff
@@ -1,12 +1,12 @@
-# Wallet configuration
 [wallet]
 
-# Connection to Rusk
 [rusk]
 ipc_method = "tcp_ip"
-rusk_addr = "http://127.0.0.1:8585"
-prover_addr = "http://127.0.0.1:8585"
+rusk_addr = "https://nodes.dusk.network:8585"
+prover_addr = "http://nodes.dusk.network:8686"
```

Comment:
This should be http**s**://**provers**.dusk.network:8686

Reply:
https://github.com/dusk-network/rusk/pull/671/commits/15434ab38cbf44b1d2d03902db5d7549a87ea0f2